#[test]
fn fail_test() {
    //panic!("I'm failing!");
}

#[test]
fn fail_test2() {
    //Test that the cache is working
}
